my_dict_2 = {'name':'ahmad','number':123456}

print(my_dict_2['name'])  # output is ahmad

print(my_dict_2.get('name'))  # output is ahmad

# print(my_dict_2['address'])  # it raises keyError
print(my_dict_2.get('address'))  # it returns None



